docs/integrations/saas-cloud/ms-graph-azure-ad-reporting.md

```diff
@@ -0,0 +1,333 @@
+| where category matches "{{resource_category}}"
+| where logged_by_service matches"{{service}}"
+| where operation_result matches "{{operation_result}}"
+| where operation_type matches "{{operation_type}}"
+| if (isNull(is_user_initiator),"app","user") as initiator
+| where initiator matches "{{initiator}}"
+| count_distinct(id) as frequency by category
+| sort by frequency
+```
+
+```sql title="Sign-In Over Time (Sign-In Acitvity)"
+_sourceCategory="azure_ad_reporting" "appDisplayName" "clientAppUsed" "ipAddress" "resourceId"
+| json "id","ipAddress","clientAppUsed","isInteractive","resourceDisplayName","riskDetail","riskEventTypes","riskLevelAggregated","riskState","status.failureReason","conditionalAccessStatus" as id, ip,client_app_used,is_interactive,resource,risk_reason,risk_event_types,risk_level,risk_state,failure_reason,conditional_activity_status nodrop
+| where risk_reason matches "{{risk_reason}}"
+| where risk_level matches "{{risk_level}}"
+| where risk_state matches "{{risk_state}}"
+| where client_app_used matches "{{client_app_used}}"
+| where conditional_activity_status matches "{{conditional_access_policy_status}}"
+| if(isNull(failure_reason) or isBlank(failure_reason),"false","true") as sign_in_failed
+| where sign_in_failed matches "{{sign_in_failed}}"
+| timeslice 1d
+| count_distinct(id) by _timeslice
+| fillmissing timeslice
+```
+
+```sql title="Average Provisioning Activity Time (Provisioning Acitvity)"
+_sourceCategory="azure_ad_reporting" "provisioningAction" "provisioningSteps" "provisioningStatusInfo"
+| json "id","provisioningStatusInfo.status","provisioningAction","durationInMilliseconds","initiatedBy.initiatingType","servicePrincipal.displayName","sourceIdentity.identityType","sourceSystem.displayName","targetIdentity.identityType","targetSystem.displayName" as id,provisioning_status,provisioning_action,duration_in_ms,initiated_by,service_principal,source_identity_type,source_system,target_identity_type,target_system_name nodrop
+| where initiated_by matches "{{initiated_by}}"
+| where provisioning_action matches "{{provisioning_action}}"
+| where provisioning_status matches "{{provisioning_status}}"
+| where service_principal matches "{{service_principal}}"
+| count_distinct(id) by duration_in_ms
+| avg(duration_in_ms)
+```
+</details>
+
+## Setup Collection
+
+Follow the instructions for setting up [Microsoft Graph Azure AD Protection](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/ms-graph-azure-ad-reporting-source/) source and use the same source category while installing the app.
+
+## Installing the Microsoft Graph Azure AD Protection app​
+
+This section has instructions for installing the Microsoft Graph Azure AD Protection app for Sumo Logic and descriptions of each of the dashboards.
```

Comment:
```suggestion
This section has instructions for installing the Microsoft Graph Azure AD protection app for Sumo Logic and descriptions of each of the dashboards.
```

Reply:
Changed as "Reporting". And the letter R is in caps as it is a part of app name.